test/unit/local_dependency/ModuleCMakeLists.txt.in

```diff
@@ -32,3 +32,4 @@ include(@CPM_PATH@/testing.cmake)
 ASSERT_TRUTHY(@TEST_DEPENDENCY_NAME@_ADDED)
 ASSERT_DEFINED(@TEST_DEPENDENCY_NAME@_SOURCE_DIR)
 ASSERT_DEFINED(@TEST_DEPENDENCY_NAME@_BINARY_DIR)
+ASSERT_EQUAL(CPM_LAST_PACKAGE_NAME "@TEST_DEPENDENCY_NAME@")
```

Comment:
I believe the variable should be expanded here and the other tests below.

```suggestion
ASSERT_EQUAL("${CPM_LAST_PACKAGE_NAME}" "@TEST_DEPENDENCY_NAME@")
```

Reply:
Agreed, much better 💯

Reply:
Agree. Will look into these changes.

Reply:
Done